src/lib.rs

```diff
@@ -85,6 +85,7 @@ macro_rules! directory_entry(
 
 directory_entry!(ExportTable         = RVA<ExportDirectory>);
 directory_entry!(BaseRelocationTable = RVA<RelocationBlock>);
+directory_entry!(Debug               = RVA<DebugDirectory>);
```

Comment:
I think that's actually too strict a limitation and we can switch to `pub trait Directory where Self::Type: RefSafe`